front-end/src/domains/game-view/components/trade-modal/trade-modal.component.tsx

```diff
@@ -0,0 +1,28 @@
+import { TrayData } from "../icon-tray";
+import "./trade-modal.style.scss";
+
+type TradeModalProps = {
+    actionsMap: (action: TrayData|undefined) => void;
+    itemData?: TrayData;
+};
+export const TradeModal: React.FC<TradeModalProps> = ({ actionsMap, itemData }) => (
```

Comment:
If you want to strip out the base React imports, you can just import the FC type from the library like this:

`import { FC } from 'react';`